external/fv3fit/fv3fit/keras/_models/models.py

```diff
@@ -317,9 +318,7 @@ def dump(self, path: str) -> None:
             with open(os.path.join(path, self._Y_SCALER_FILENAME), "wb") as f_binary:
                 self.y_scaler.dump(f_binary)
             with open(os.path.join(path, self._OPTIONS_FILENAME), "w") as f:
-                yaml.safe_dump(
-                    {"normalize_loss": self._normalize_loss, "loss": self._loss}, f
-                )
+                yaml.safe_dump(dataclasses.asdict(self._hyperparameters), f)
```

Comment:
This is great to be able to dump this estimator's hyperparameters in a self-defining way.